sos/report/plugins/aap_controller.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2023 Pavel Moravec <pmoravec@redhat.com>
+# Copyright (c) 2024 Lucas Benedito <lbenedit@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPControllerPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP Automation Controller plugin'
+    plugin_name = 'aap_controller'
+    profiles = ('sysmgmt', 'ansible')
+
+    packages = ('automation-controller-venv-tower',
+                'automation-controller-server',
+                'automation-controller-ui',
+                'automation-controller')
+    commands = ('awx-manage')
+    services = ('automation-controller')
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/tower/",
+            "/etc/receptor/",
+            "/etc/supervisord.conf",
+            "/etc/supervisord.d/*",
+            "/var/log/tower",
+            "/var/log/nginx/automationcontroller.access.log*",
+            "/var/log/nginx/automationcontroller.error.log*",
+            "/var/log/supervisor",
+            "/var/log/unattended-upgrades",
+        ])
+
+        self.add_forbidden_path([
+            "/etc/tower/SECRET_KEY",
+            "/etc/tower/tower.key",
+            "/etc/tower/awx.key",
+            "/etc/tower/tower.cert",
+            "/etc/tower/awx.cert",
+            "/var/log/tower/profile",
+            "/etc/receptor/tls/ca/*.key",
+            "/etc/receptor/tls/*.key",
+        ])
+
+        self.add_cmd_output([
+            "awx-manage --version"
+            "awx-manage list_instances"
+            "awx-manage run_dispatcher --status"
+            "awx-manage run_callback_receiver --status"
+            "awx-manage check_license --data"
+            "awx-manage run_wsbroadcast --status"
+            "awx-manage run_wsrelay --status",
+            "supervisorctl status"
+            "/var/lib/awx/venv/awx/bin/pip freeze"
+            "/var/lib/awx/venv/awx/bin/pip freeze -l"
+            "/var/lib/awx/venv/ansible/bin/pip freeze"
+            "/var/lib/awx/venv/ansible/bin/pip freeze -l"
+            "tree -d /var/lib/awx"
+            "ls -ll /var/lib/awx"
+            "ls -ll /var/lib/awx/venv"
+            "ls -ll /etc/tower",
+            "ls -ll /var/run/awx-receptor"
+            "ls -ll /etc/receptor",
+            "receptorctl --socket /var/run/awx-receptor/receptor.sock status"
```

Comment:
Aren't these collected by the plugin proposed in #3553?

Reply:
Ack, will remove the receptor references. thanks